.github/ISSUE_TEMPLATE/intent-to-implement--i2i-.md

```diff
@@ -15,6 +15,11 @@ See https://github.com/ampproject/amphtml/blob/master/contributing/contributing-
 If you haven't already done so, sign the Contributor License Agreement (CLA) as soon as possible to avoid delays merging your code. A signed CLA is not necessary to submit this I2I or to send a pull request, but it will be needed before your code can be merged. See https://github.com/ampproject/amphtml/blob/master/contributing/contributing-code.md#contributor-license-agreement for more information on CLAs.
 -->
 
+## Status
+<!--
+Draft, Ready for Review
```

Comment:
Make it more explicit that the value someone specifies in this section should be one of these two values. Also, have some explanatory text explaining the purpose of creating the issue in either of these cases. I think both of these could be accomplished with something like:
```
- [ ] Ready for review

<!--
Issues that aren't marked "Ready for review" are considered drafts. Draft I2Is are encouraged for particularly big or fundamental proposals in order to promote broader community awareness and get earlier feedback.
-->
```